ppml/base/entrypoint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
is this the entrypoint to all the sub-images?
we should keep the pccs url as ENV and can be set at runtime, the enclave key signing process requires a unchanged file in the image